jekyll/_cci2/testing-orbs.md

```diff
@@ -8,178 +8,170 @@ order: 1
+Remember, including bats tests are optional and can be removed from your configuration file if desired.
+{: class="alert alert-warning"}
 
-Yet another approach is when you run a build, publish a dev version of the orb, then do an automated push to a branch which uses a config that uses that dev version. This new build can run the tests. The downside with this approach is that the build that does the real testing is not directly tied to its commit, and you end up with multiple jobs every time you want to change your orb.
 
-### Integration Testing
+## Integration Testing
 
-Sometimes you will want to test how your orbs interact with external services. There are several possible approaches depending on circumstances:
+Up until this point all testing had occurred prior to packing the orb, and applied to the code itself but not the finalized functionality of our orb. For the final critical part of orb testing, we want to actually test our orb commands and jobs to ensure they work as intended in production. We do this after we have ran our validation tests and published a new development version of the orb which we will use to test.
 
-* Make your orb support a dry-run functionality of whatever it is interacting with, and use that mode in your tests.
-* Do real interactions, using a properly set up test account and a separate repository that runs those tests using a published dev version of your orb.
-* Spin up a local service in another container of your job.
+After the development version of the orb has been published, the [integration-test_deploy](https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/.circleci/config.yml#L78) workflow will be automatically triggered and use the newly created development version of the orb.
 
-## Orb Testing Best Practices
+Within the `integration-test_deploy` workflow, we run a series of final integration tests, and if all pass, and we are on our main deployment branch, then we may also deploy our orb.
 
-The most significant issue when testing orbs is that it is not straightforward to push a new commit to a repository containing orb source code. This is because orbs are interpolated into an expanded `config.yml` at build inception and may not have the newest changes to the orb contained in that commit.
+### How To Test Orb Commands
 
-There are several different approaches you can use to test your orbs (for example, using inline orbs or external repositories) to ensure orb compatibility with the CircleCI platform. CircleCI is also in the process of developing newer ways for you to test your orbs.
+The first job you will see in the [integration-test_deploy](https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/.circleci/config.yml#L78) workflow is the [integration-test-1](https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/.circleci/config.yml#L82) job, a sample integration test included with the `hello-world` orb generated by the `orb-init` command.
```

Comment:
It's not immediately clear to me what we're suggesting here - do we want people to go and edit this job to run the commands included in their own orb of there are any?

Reply:
We do 👍 . ill see if I can play with this